FakeServer/Program.cs

```diff
@@ -122,15 +123,31 @@ private static int Run(string[] args, Dictionary<string, string> initialData)
                     app.RemainingArguments.Add(optionsUrls.Value());
                 }
 
+                if (optionInit.HasValue()) {
```

Comment:
Use same formatting as the rest of the code base. Move curly brackets to new line.